snapcraft/internal/lifecycle/_clean.py

```diff
@@ -176,6 +171,7 @@ def clean(project_options, parts, step=None):
 
     config = project_loader.load_config()
 
+    # FIXME: SHOULDN'T THIS DO THE SAME IF STEP < stage?
```

Comment:
it would be easy to parse if we had an ordered enum (a list, oh we do), but that index thing is not as easy to read. Feel free to change given that we might add more steps.

Reply:
Yeah I agree, that index thing works nicely but it's rough to read. I'll see if I can wrap it in at least a function or enum to improve readability.

Reply:
This is in #2159.